cmake/Modules/FindAzureCore_EP.cmake

```diff
@@ -0,0 +1,163 @@
+#
+# FindAzureCore_EP.cmake
+#
+#
+# The MIT License
+#
+# Copyright (c) 2018-2021 TileDB, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+# This module finds the Azure core SDK, installing it with an ExternalProject if
+# necessary. It then defines the imported by target Azure::azure-core.
+
+# Include some common helper functions.
+include(TileDBCommon)
+
+# Azure dependencies
+if (TILEDB_VCPKG)
+    return()
+endif()
+
+###############################################################################
+# Start superbuild/unmanaged/legacy version
+###############################################################################
+
+# This is the install subdirectory for azure headers and libs, to avoid pollution
+set(TILEDB_EP_AZURE_INSTALL_PREFIX "${TILEDB_EP_INSTALL_PREFIX}/azurecpplite")
```

Comment:
Why do we put them to a separate directory? I'm going to revert it.

Reply:
Well there is a lot of pollution of `include`, so I left it in the separate place. I don't think it will effect things in the new sdk so give it a try!